corehq/apps/groups/templates/groups/group_members.html

```diff
@@ -92,53 +103,120 @@
         <!-- Tab panes -->
         <div class="tab-content">
             <div role="tabpanel" class="tab-pane" id="membership-tab">
-                <div class="row">
-                    <div class="col-sm-12">
-                        <div class="alert alert-info" id="membership_updating" style="display: none">
-                          <p class="lead">
-                            <i class="fa fa-spinner fa-spin"></i> {% trans 'Updating Group Membership, please wait...' %}
-                          </p>
+                <div class="panel-body">
+                    {% if request.is_view_only or not can_edit_group_membership %}
+                        <div class="form form-horizontal">
+                            <legend>
+                                {% trans "View Group Membership" %}
+                            </legend>
+                            <div class="form-group">
+                                <label class="control-label col-xs-12 col-sm-4 col-md-4 col-lg-2">
+                                    {% trans "Group Membership" %}
+                                </label>
+                                <div class="col-xs-12 col-sm-8 col-md-8 col-lg-6 controls-text">
+                                    {% if not group_members %}
+                                        {% blocktrans %}
+                                            No Mobile Workers are currently assigned to this group.
+                                        {% endblocktrans %}
+                                    {% else %}
+                                        <ul>
+                                            {% for member in group_members %}
+                                                <li>{{ member.username_in_report }}</li>
+                                            {% endfor %}
+                                        </ul>
+                                    {% endif %}
+                                </div>
+                            </div>
                         </div>
+                        {% if not can_edit_group_membership %}
+                            <div class="alert alert-info">
+                                {% blocktrans %}
```

Comment:
I set a user to have permission to "view groups" only and not "edit groups" but ability to "edit commcare users", but this message says "Your permissions allow you to create, update, and delete groups", which is not true so this needs to be updated?

Reply:
@mkangia i think this was, again, work that was out of scope of this ticket. we didn't perfect the permissions on this first round with an understanding further work was still needed.